superset/commands/report/execute.py

```diff
@@ -139,22 +139,37 @@ def update_report_schedule_slack_v2(self) -> None:
                     slack_recipients = json.loads(recipient.recipient_config_json)
                     # we need to ensure that existing reports can also fetch
                     # ids from private channels
+                    channels_list = get_recipients_list(slack_recipients["target"])
+                    channels_list = [channel.lstrip("#") for channel in channels_list]
+                    channels = get_channels_with_search(
+                        search_string=channels_list,
+                        types=[
+                            SlackChannelTypes.PRIVATE,
+                            SlackChannelTypes.PUBLIC,
+                        ],
+                        exact_match=True,
+                    )
+                    if len(channels_list) != len(channels):
+                        missing_channels = set(channels_list) - {
+                            channel["name"] for channel in channels
+                        }
+                        msg = (
+                            "Could not find the following channels: "
+                            f"{', '.join(missing_channels)}"
+                        )
+                        raise UpdateFailedError(msg)
+                    channel_ids = ", ".join(channel["id"] for channel in channels)
```

Comment:
nit, do we need the space after the comma here?
```suggestion
                    channel_ids = ",".join(channel["id"] for channel in channels)
```

Reply:
Oh, we don't. Removing this now.